packages/op-datasets/src/op_datasets/etl/intermediate/models/daily_address_summary.py

```diff
@@ -211,27 +212,26 @@
 
 @register_model
 def daily_address_summary(
-    task: IntermediateModelsTask,
-) -> dict[str, duckdb.DuckDBPyRelation]:
-    txs: duckdb.DuckDBPyRelation = task.input_duckdb_relations["transactions"]
-    blks: duckdb.DuckDBPyRelation = task.input_duckdb_relations["blocks"]
-
-    client = init_client()
+    duckdb_client: duckdb.DuckDBPyConnection,
```

Comment:
slight change, but model functions now require that the duckdb_client is passed into them. This makes it easier to use them in tests.

Reply:
Does this structure apply to future intermediate tables?

Reply:
Yes! for new models the function signature will have those two parameters.  There is also now a requirement to be more clear about what data does a model consume. You don't see it yet in this PR, but in the main branch it now looks like this:
```
@register_model(
    input_datasets=["blocks", "transactions"],
    expected_outputs=["daily_address_summary_v1"],
)
def daily_address_summary(
    duckdb_client: duckdb.DuckDBPyConnection,
    input_tables: NamedRelations,
) -> NamedRelations:
```

So here the `daily_address_summary` function is producing results that will be stored in gcs under a versioned name: `daily_address_summary_v1`.  This will allow us to decouple the name of the function from where we want to  store the results.

Reply:
Would like to see some results when it's available in some test run whenever

Reply:
definitely, working on that as we speak, hoping to show you some data in an actual table by tomorrow